liferay-echo/liferay-portal
# LPS-194942 Add back title to all back buttons in edit mode for layouts

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-194942)

## Proposed solution

Add back title to all back button in edit mode for layouts. That means, adding it in the dropdown, when clicking in the name of the vertical card and when adding a new one. Also there are especial cases, defined below.

## Test Scenario 1 

We need to check that for every type, when access to the edit mode from dropdown, clicking on the name of the card or when adding a new one, the URL includes the `“p_l_back_url_title”` parameter so the back button says “Go to x” where x could be Page Templates or Pages  depending of the case/type.

https://github.com/liferay-echo/liferay-portal/assets/93203423/702871fb-fe85-47dd-a5eb-29cfca7316db

## Test Scenario 2

1. Create a Web Content and add a Specific Display Page Template for it.
2. Publish it and then choose “View Content”.
3. Click on the pencil to edit an select “Edit Display Page Template”
4. Check that the tooltip should be the "Go to" and the Web Content Title.

<img width="355" alt="image" src="https://github.com/liferay-echo/liferay-portal/assets/93203423/083b1f66-9f8b-4e8d-ba7c-b6681e27a51e">

## Test Scenario 3

1. Create a new Master Page.
2. Create a new Content Page. 3. Go to the configuration in “Page Design Option” of the page and select the master page created in step 1. Save.
4. Click on the edit button next to the new Master Page assigned. 

https://github.com/liferay-echo/liferay-portal/assets/93203423/b1f64438-8f2d-4b68-985c-640173165213

* Note: this behaviour has been fix. Now the expected behaviour is going back to design settings when clicking the back button.
